test/spec/modules/intentIqIdSystem_spec.js

```diff
@@ -492,4 +493,18 @@ describe('IntentIQ tests', function () {
     const savedClientHints = readData(CLIENT_HINTS_KEY, ['html5']);
     expect(savedClientHints).to.equal(handleClientHints(testClientHints));
   });
+
+  it('should get firstPartyData', async () => {
```

Comment:
The unit test name is incorrect, and the test itself needs to be updated. Previously, there was a bug where, if the browser was on the blacklist, it couldn't read first-party data. This caused the data not to be generated or read, resulting in an error and the callback not being triggered. In reality, there shouldn't be any first-party data, but the callback should still be triggered with `runtimeEids` (empty object) and the group 'U' (undefined).